integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -63,24 +65,116 @@ If this does not suit your needs and you want to experiment with different promp
+
+Follow the steps in the following sections for the custom extensions setup before uploading the sample action.
+
+### Configure the Hugging Face Hub Embeddings extension
+
+1. Sign up at [Hugging Face](https://huggingface.co/) and follow the instructions there to get an API key. Save this API key somewhere safe and accessible.
+
+1. Download the OpenAPI specification file [hf-hub-embed.json](hf-hub-embed.json) from this starter kit. You use this specification file to create and add the Hugging Face Hub Embeddings extension to your assistant.
+
+1. In your assistant, on the **Integrations** page, click **Build custom extension** and use the OpenAPI specification file to build a custom extension named `Hugging Face Hub Embeddings`. For general instructions on building any custom extension, see [Building the custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
+
+1. After you build the Hugging Face Hub Embeddings extension and it appears on your **Integrations** page, click **Add** to add it to your assistant. For general instructions on adding any custom extension, see [Adding an extension to your assistant](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-add-custom-extension).
+
+1. In **Authentication**, choose **Bearer auth**. Copy and paste your Hugging Face API key into the **Token** field.
+
+1. The setup of your extension is now complete. For more information about the extension parameters, see the [Hugging Face API Reference](https://huggingface.co/docs/hub/api).
+
+### Configure the Milvus semantic search extension
+
+Milvus is an open-source vector database for semantic search.
+
+1. Follow the instructions at [Milvus](https://milvus.io) to [install it](https://milvus.io/docs/install_standalone-docker.md) somewhere that your assistant can access it. You will need the server variables to connect it with your assistant. One option for where to host is an [IBM virtual server](https://www.ibm.com/cloud/virtual-servers). Another is [IBM code engine](https://www.ibm.com/cloud/code-engine). However, it can also be installed on a separate machine as long as it is connected to the internet. Alternatively, Milvus can be run on a different cloud such as AWS, in which case you need to record the IP address of the machine where it's installed Milvus and the port where it is running.
+
+1. Add your content to Milvus. In this example, we use Milvus to search a document collection that we previously indexed with [this python script](index-with-milvus.py). We include this script as an example for reference. In our example we use `.pdf` documents; the script can be modified to use other text formats.
```

Comment:
I would also point out that we're not including the sample documents because we encourage you to use your own and modify the script accordingly.